app/services/usps_in_person_proofing/mock/responses/request_expired_proofing_results_response.json

```
@@ -1,15 +1,15 @@
 {
-    "status": "In-person expired",
-    "proofingPostOffice": "WILKES BARRE",
-    "proofingCity": "WILKES BARRE",
-    "proofingState": "PA",
-    "enrollmentCode": "2090002197604352",
-    "primaryIdType": "Uniformed Services identification card",
-    "transactionStartDateTime": "12/17/2020 033855",
-    "transactionEndDateTime": "12/17/2020 034055",
-    "secondaryIdType": "Deed of Trust",
-    "responseMessage": "More than 30 days have passed since opt-in to IPP",
-    "fraudSuspected": false,
-    "proofingConfirmationNumber": "350040248346707",
-    "ippAssuranceLevel": "1.5"
-}
\ No newline at end of file
+  "status": "In-person expired",
+  "proofingPostOffice": "WILKES BARRE",
+  "proofingCity": "WILKES BARRE",
+  "proofingState": "PA",
+  "enrollmentCode": "2090002197604352",
+  "primaryIdType": "Uniformed Services identification card",
+  "transactionStartDateTime": "12/17/2020 033855",
+  "transactionEndDateTime": "12/17/2020 034055",
+  "secondaryIdType": "Deed of Trust",
+  "responseMessage": "More than 30 days have passed since opt-in to IPP",
+  "fraudSuspected": false,
+  "proofingConfirmationNumber": "350040248346707",
+  "ippAssuranceLevel": "1.5"
+}
```

Per [this comment](https://github.com/18F/identity-idp/pull/6967#discussion_r977733243) I suggest deleting most of these fields because USPS doesn't actually return them for expired enrollments

```suggestion
  "responseMessage": "More than 30 days have passed since opt-in to IPP"
}
```